src/helpers/check-special-chars-and-empty.js

```diff
@@ -1,8 +1,14 @@
-export const checkSpecialCharsAndEmpty = (value) => {
+const SYMBOLS = '\\n\\t,;.\'"`´ ';
```

Comment:
@aichbauer If we want to use standard for quotes, how about to use only symbols from standard plus separator symbol?

Reply:
@webdevium this is definitely something we should do...

When do we actually have to use double quotes?

Do we need to escape a comma if we have a different separator? 

Maybe:

- double quote (but we have to escape a double quote inside with a leading double quote e.g. "this is a ""test""")
- separator (comma or the used separator for this document (so a comma is not escaped if there is a different separator))
- new lines

What do you think?

Reply:
I will try, but i not understand what I should do with double quote. If on symbol and if few symbols. Can you explain me with some examples, please!

Reply:
So, I did everything I could do.
Check my work, please.